includes/blocks/class-sensei-course-blocks.php

```diff
@@ -86,7 +83,7 @@ public function enqueue_block_editor_assets() {
 			return;
 		}
 
-		Sensei()->assets->enqueue( 'sensei-blocks', 'blocks/index.js', [], true );
+		Sensei()->assets->enqueue( 'sensei-course-blocks', 'blocks/course-blocks.js', [], true );
```

Comment:
We could go with `single-course` for the name, if it makes sense that these are grouped based on where they need to be loaded. The CSS files are named like that right now.

Reply:
Thank you for your review, @yscik!

I got your point, and I think it makes sense. But I think we could make this change in a future PR (maybe when we need that), just to avoid conflicts with this now: https://github.com/Automattic/sensei/pull/3885#issuecomment-758826879

It's because we'd need also to move this file out of the `blocks` folder to make sense.